node/compatibility.md

```diff
@@ -0,0 +1,357 @@
+
+ℹ️ Partially polyfilled
+
+Missing `utf16le`, `latin1` and `ucs2` encoding for `fs.writeFile` and
+`fs.writeFileSync`. Missing `Dirent.isBlockDevice`, `Dirent.isCharacterDevice`,
+`Dirent.isFIFO`, `Dirent.isSocket`, `FSWatcher.ref`, `FSWatcher.unref`.
+
+### [`node:fs/promises`](https://nodejs.org/api/fs.html)
+
+ℹ️ Partially polyfilled
+
+Missing `lchmod`, `lchown`, `lutimes`.
+
+### [`node:http`](https://nodejs.org/api/http.html)
+
+ℹ️ Partially polyfilled
+
+`createConnection` option is currently not supported.
+
+### [`node:http2`](https://nodejs.org/api/http2.html)
+
+ℹ️ Partially polyfilled
+
+Partially polyfilled, major work in progress to enable `grpc-js`.
+
+### [`node:https`](https://nodejs.org/api/https.html)
+
+ℹ️ Partially polyfilled
+
+Missing `https.Server.opts.cert` and `https.Server.opts.key` array type
+
+### [`node:inspector`](https://nodejs.org/api/inspector.html)
+
+ℹ️ Partially polyfilled
+
+`console` is supported.
+
+Other APIs are stubs and will throw an error. Due to security implications the
+Deno team does not plan to polyfill these APIs.
+
+### [`node:module`](https://nodejs.org/api/module.html)
+
+✅ Fully polyfilled
+
+### [`node:net`](https://nodejs.org/api/net.html)
+
+ℹ️ Partially polyfilled
+
+Missing `net.Socket.prototype.constructor` with `fd` option
+
+### [`node:os`](https://nodejs.org/api/os.html)
+
+✅ Fully polyfilled
+
+### [`node:path`](https://nodejs.org/api/path.html)
+
+✅ Fully polyfilled
+
+### [`node:path/posix`](https://nodejs.org/api/path.html)
+
+✅ Fully polyfilled
+
+### [`node:path/win32`](https://nodejs.org/api/path.html)
+
+✅ Fully polyfilled
+
+### [`node:perf_hooks`](https://nodejs.org/api/perf_hooks.html)
+
+ℹ️ Partially polyfilled
+
+Missing `perf_hooks.eventLoopUtilization`, `perf_hooks.timerify`,
+`perf_hooks.monitorEventLoopDelay`.
+
+### [`node:process`](https://nodejs.org/api/process.html)
+
+ℹ️ Partially polyfilled
+
+Missing `disconnect`, `message`, `multipleResolves`, `rejectionHandled` and
+`worker` events.
+
+### [`node:punycode`](https://nodejs.org/api/punycode.html)
+
+✅ Fully polyfilled
+
+### [`node:querystring`](https://nodejs.org/api/querystring.html)
+
+✅ Fully polyfilled
+
+### [`node:readline`](https://nodejs.org/api/readline.html)
+
+✅ Fully polyfilled
+
+### [`node:repl`](https://nodejs.org/api/repl.html)
+
+ℹ️ Partially polyfilled
+
+`builtinModules` and `_builtinLibs` are supported.
+
+Missing `REPLServer.prototype.constructor` and `start()`.
+
+### [`node:stream`](https://nodejs.org/api/stream.html)
+
+✅ Fully polyfilled
+
+### [`node:stream/promises`](https://nodejs.org/api/stream.html)
+
+✅ Fully polyfilled
+
+### [`node:stream/web`](https://nodejs.org/api/stream.html)
+
+ℹ️ Partially polyfilled
+
+### [`node:string_decoder`](https://nodejs.org/api/string_decoder.html)
+
+ℹ️ Partially polyfilled
+
+Missing decoding of `ascii`, `latin1` and `utf16le` decoding options
+
+### [`node:sys`](https://nodejs.org/api/sys.html)
```

Comment:
https://nodejs.org/api/sys.html This page doesn't exist (probably because the module is deprecated now in favor of `node:util`)

Reply:
TIL, didn't know that.